nil/getting-started/prerequisites.mdx

```diff
@@ -0,0 +1,75 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Prerequisites
+
+This page details the steps that need to be completed before working with =nil;.
+
+## Installation
+
+=nil; distributed as a standalone binary. Execute the following command to install it:
```

Comment:
You probably mean that "=nil; CLI" is distributed as a standalone binary. The server itself is not yet distributed in any way.

Reply:
Actually nevermind. This is a different document